src/pages/position/create/index.jsx

```diff
@@ -0,0 +1,186 @@
+                                    <CardContainer>
+                                        <InputContainer column extramargin>
+                                            <InputLabel small>Stillingstittel</InputLabel>
+                                            <InputElement {...register("name", {required: true})} type="text" value={name} onChange={(e) => setName(e.target.value)} />
+                                        </InputContainer>
+                                    </CardContainer>
+                                </InnerContainerRow>
+
+                                <InnerContainerRow nowrap>
+                                    <CardContainer>
+                                        <InputContainer column extramargin>
+                                            <InputLabel small>Stillingsbeskrivelse</InputLabel>
+                                            <InputElement {...register("description", {required: true})} type="text" value={description} onChange={(e) => setDescription(e.target.value)} />
+                                        </InputContainer>
+                                    </CardContainer>
+                                </InnerContainerRow>
+                            </InnerContainer>
+                            
+                            <InnerContainer flex="1" floattop rowgap>
+                                <InnerContainer>
+                                    <InnerContainerTitle>Tilknytning til crew og team</InnerContainerTitle>
+                                    <InnerContainerRow norowgap nowrap>
+                                        <CardContainer>
+                                            <InputContainer column extramargin>
+                                                <InputLabel small>Tilknytt crew</InputLabel>
+                                                <InputSelect {...register("crew_uuid")} value={attachedCrew} onChange={(e) => setAttachedCrew(e.target.value)}>
+                                                    <option value={null} label="Ingen tilknytning" />
+                                                    {
+                                                    crewList.map((crew) => (<option key={crew.uuid} value={crew.uuid}>{crew.name}</option>))
+                                                    }
+                                                    
+                                                </InputSelect>
+                                            </InputContainer>
+                                        </CardContainer>
+                                        <CardContainer>
+                                            <InputContainer column extramargin>
+                                                <InputLabel small>Tilknytt team</InputLabel>
+                                                <InputSelect {...register("team_uuid")}>
+                                                    <option></option>
```

Comment:
The phoenix UI still does not really support teams, so no reason to show teams IMO. It just makes things more confusing